### PR TITLE
Completes DELETE /api/comments/:comment_id

### PR DESCRIPTION
**Pull request by danrfletcher**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -140,4 +140,40 @@ describe('GET Requests', () => {
...
+});
+
+describe('DELETE Requests', () => {
+    describe('DELETE /api/comments/:comment_id', () => {
+        test('/api/comments/:comment_id 200: responds with deleted comment when ID is valid', () => {
+            return request(app)
+            .delete('/api/comments/1')
+            .expect(200)
+            .then((res) => {
+                expect(res.body.comment).toEqual(expect.objectContaining({
```
Comment (nc-hev-mag) [inline __tests__/app.test.js]: This is great if you want to send back the deleted comment to the user - you could alternatively just send back a 204 status which doesn't (no content) but does have a more descriptive status code. Depends on what you want your user experience to be

File: models/comment-models.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection.js');
+const format = require('pg-format');
+
+exports.deleteComment = (comment_id) => {
+    return db.query(format(`DELETE FROM comments WHERE comment_id = %L RETURNING *;`, comment_id))
```
Comment (nc-hev-mag) [inline models/comment-models.js:5]: %L is normally used for nested arrays in my experience so $1 syntax would probs be more suitable here but it's up to you, both accomplishing the same thing just worth noting!

**Review by nc-hev-mag — APPROVED** [review]
Nice one Dan - couple of notes for your consideration but looks good to merge 👍

File: models/comment-models.js (hunk 2 above)
Comment (danrfletcher) [reply models/comment-models.js:5]: Thanks Hev, not sure what's going wrong with the $1 syntax but I haven't been able to get it to work?!

**Pull request merged**